Gemfile

```
@@ -11,5 +11,6 @@ elsif ENV['RAILS_VERSION']
 end
 group :development do
   # test/unit is no longer bundled with Ruby 2.2, but required by Rails
+  gem "pry"
```

Probably ought to be `require: false` so it's not loaded when not needed
```suggestion
  gem "pry", require: false
```